tests/Unit/Domain/Asset/AssetTest.php

```
@@ -8,43 +8,163 @@
+        $createdAt = new DateTimeImmutable('2026-01-20T12:34:56+00:00');
+
+        // Act
+        $asset = Asset::reconstituteUploaded(
+            $assetId,
+            $uploadId,
+            $accountId,
+            '  ' . self::FILE_NAME . '  ',
+            '  ' . self::MIME_TYPE . '  ',
+            $createdAt,
+            $this->createCompletionProofValue(),
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Domain\Asset\Asset' in method 'itReconstitutesUploadedAssetWithPersistedValues'. (undefined)

(StaticAccess)